app/assets/stylesheets/comments.scss

```diff
@@ -966,27 +933,26 @@ a.header-link {
 
       .dropdown-content {
         display: none;
+        font-size: var(--fs-base);
         position: absolute;
         right: 0;
-        border: 1px solid $light-medium-gray;
         border-radius: 3px;
-        background: white;
+        @include generate-box(
+          $level: 2,
+          $bg: var(--card-bg),
+          $border: var(--card-color),
+          $color: var(--card-color)
+        );
         z-index: 20;
-        width: 130px;
-        box-shadow: $shadow;
+        width: 160px;
```

Comment:
Just a general question here in regards to width. Why aren't we using `10rem`?

Reply:
I haven't thought about it and I have no idea... But thinking about it right now... If we ever change the font size a little bit, do we also want to adjust the layout of some elements (since rem units are based on font size). But I don't have any strong opinions there yet, I'd have to google it a bit more.

Another thing is that it's easier to visualize what "160px" mean instead of "10 rem" mostly because you don't have to do the math.

Lemme know if you have any thoughts or opinions there.

Reply:
I was thinking more about if people zoom. `rem` and `em` help with that.

Reply:
As I mentioned it's not specific to this PR, just more of a general question.

Reply:
Definitely worth talking about - I'm not very experienced with accessibility and I don't think I know exactly how zooming works.

But that may seem indeed as bigger discussion issue to be solved in separate PR..